package.json

```diff
@@ -15,7 +15,8 @@
   "exports": {
     ".": {
       "import": "./lib/esm/index.js",
-      "require": "./lib/cjs/index.js"
+      "require": "./lib/cjs/index.js",
+      "types": "./lib/types/index.t.ts",
```

Comment:
I believe it should be: `"types": "./lib/types/index.d.ts"`

Reply:
Will correct this before publishing 👍

Reply:
Hey!
I just caught you merging this right now and I am here to correct my mistake. Now I know better.

The `types` should be outside of `exports`, in the main level of `package.json`. It also can (not sure if should) have an absolute path (no `./` at the beginning).

So finally: `"types": "lib/types/index.t.ts",` on the main level.

Sorry for this!